test/e2e/admin_portal.go

```diff
@@ -134,6 +134,86 @@ var _ = Describe("Admin Portal E2E Testing", func() {
 		}
 	})
 
+	It("Should be able to populate feature flag panel correctly", func() {
+		const (
+			CLUSTER_FEATURE_HEADINGS = 24
```

Comment:
I would use lowercase here as it's the most adopted style for constants in Go.

Reply:
Done, fixed it!
Fixed the same in another test also for the cluster info panel